python/cuspatial/cuspatial/core/binpreds/feature_contains.py

```diff
@@ -132,20 +132,25 @@ def _compute_predicate(self, lhs, rhs, preprocessor_result):
 
 class PointPointContains(BinPred):
     def _preprocess(self, lhs, rhs):
-        return _basic_equals(lhs, rhs)
+        return _basic_equals_any(lhs, rhs)
 
 
 class LineStringPointContains(BinPred):
     def _preprocess(self, lhs, rhs):
         intersects = _basic_intersects(lhs, rhs)
-        equals = _basic_equals(lhs, rhs)
+        equals = _basic_equals_any(lhs, rhs)
         return intersects & ~equals
 
 
 class LineStringLineStringContainsPredicate(BinPred):
     def _preprocess(self, lhs, rhs):
-        count = _basic_equals_count(lhs, rhs)
-        return count == rhs.sizes
+        # A linestring A covers another linestring B iff
```

Comment:
Is this comment correct? This is contains, not covers. The code for the two looks the same. Is that correct?

Reply:
```
import geopandas
from cuspatial.tests.binpreds.binpred_test_dispatch import (
    features,
    linestring_linestring_dispatch_list,
)
def sample_test_data(features, dispatch_list, size, lib=cuspatial):
    geometry_tuples = [features[key][1:3] for key in dispatch_list]
    geometries = [
        [lhs_geo for lhs_geo, _ in geometry_tuples],
        [rhs_geo for _, rhs_geo in geometry_tuples]
    ]
    lhs = lib.GeoSeries(list(geometries[0]))
    rhs = lib.GeoSeries(list(geometries[1]))
    if lib == geopandas:
        rng = np
    else:
        rng = cp
    rng.random.seed()
    lhs_picks = rng.random.randint(0, len(lhs), size)
    rhs_picks = rng.random.randint(0, len(rhs), size)
    return (
        lhs[lhs_picks].reset_index(drop=True),
        rhs[rhs_picks].reset_index(drop=True)
    )
lhs, rhs = sample_test_data(features, linestring_linestring_dispatch_list, 1000, geopandas)
(lhs.contains(rhs) == lhs.covers(rhs)).all()
```
```
True
```